### PR TITLE
[MVT] Fix #4561 and #4565: Apply style when advanced edit opens

### LINKED ISSUE #4561: [MVT] F-006 Third party style by (source) : you can export but not edit"
When triying edit a third party style as in https://mf-geoadmin3.int.bgdi.ch/mvt/index.html?lang=en&topic=ech&bgLayer=omt.vt&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,&bgLayer_styleUrl=https:%2F%2Fraw.githubusercontent.com%2Fopenmaptiles%2Ffiord-color-gl-style%2Fmaster%2Fstyle.json&E=2601011.88&N=1199625.93&zoom=17 

 Add in 

image

- [ ] msgid " Third party style by (source) : you can export but not edit"
- [ ] even deactivate the pulldown "Choose layer"
or
- [ ] don't even open the Advanced EDIT mode: only show msgid " Third party style by (source) : you can export but not edit"

### PR DESCRIPTION
**Pull request by oterral**
<jenkins>A test link will be added here if the deploy on int succeeded.</jenkins>

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/teo_fixes/index.html)</jenkins>

### CODE REVIEW

File: test/specs/js/EditController.spec.js [hunk 1]
```diff
@@ -62,13 +62,18 @@ describe('ga_edit_controller', function() {
         expect(scope.globals).to.be.an(Object);
       });
 
-      it('set scope values on gaToggleEdit event', function() {
+      it.only('set scope values on gaToggleEdit event', function() {
```
Comment (loicgasser) [inline test/specs/js/EditController.spec.js]: only

Comment (oterral) [reply test/specs/js/EditController.spec.js]: fixed

File: src/js/EditController.js [hunk 2]
```diff
@@ -26,13 +26,18 @@ goog.require('ga_mvt_service');
...
+      if (toggle === undefined) {
+        toggle = !!(!$scope.globals.isEditActive ||
+          !$scope.layer ||
+          (layer && layer.bodId !== $scope.layer.bodId));
+      }
       $scope.layer = layer;
       $scope.globals.isEditActive = toggle;
       $scope.globals.pulldownShown = toggle;
+      if (toggle) {
+        gaMvt.reload($scope.layer);
```
Comment (loicgasser) [inline src/js/EditController.js]: we don't want to reload... We want to apply the style attached to layer

Comment (loicgasser) [reply src/js/EditController.js]: Ok here is what I had in mind: https://github.com/geoadmin/mf-geoadmin3/pull/4580/commits/08548434a050d5026bfc844057e397f0d04ea475

**Pull request merged**